sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -78,19 +78,39 @@ Licensed under the MIT License.
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
I thought we agreed to not add dependency on these directly? Instead take them from azure-core?

Reply:
This is needed for the time being until Core releases, or I could try a change where Cosmos takes a source version on Core. Either way, after Core releases the Cosmos versions will need to be modified again.

Reply:
Thanks, makes sense now.